src/components/TodoContext.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { Todo } from '../types/Todo';
+import { Status } from '../services/EnumStatusFilter';
+import { useLocaleStorage } from '../hooks/useLocaleStorage';
+import { Context } from '../services/Context';
+
+type Props = {
+  children: React.ReactNode,
+};
+
+export const TodoContext = React.createContext<Context>({
+  todos: [],
+  setTodos: () => {},
+  addTodo: () => {},
+  count: 0,
+  title: '',
+  setTitle: () => {},
+  updateTodo: () => {},
+  checkedAll: () => {},
+  selTodoFilterList: Status.ALL,
+  setSelTodoFilterList: () => {},
+  deleteTodo: () => {},
+  deleteAllCompleted: () => {},
+});
+
+export const TodoProvider: React.FC<Props> = ({ children }) => {
+  const [todos, setTodos] = useLocaleStorage<Todo[]>('todos', []);
```

Comment:
why do you need to store todos in the Context when it's stored in LocaleStorage?